docs/readmes/README.msi.md

```diff
@@ -17,25 +17,19 @@ The rest of the README describes the prerequisite role assignments to be perform
 ## Pre-requisites - role assignments
 MIC is responsible for performing operations such as assigning user assigned identity to the underlying vm or vmss which makes up the
 nodes in the Kubernetes cluster. The system/user assigned MSI needs to have role assignments authorizing such operations on the vms/vmss
-and also operations on the user assigned identity.
+and also operations on the user assigned identities.
 
-After the cluster is created, run these commands to retrieve the principal id:
-for VMAS:
-
-```bash
-az vm identity show -g <resource group> -n <vm name> -o yaml
-```
-for VMSS:
+After the cluster is created, run this command to retrieve the cluster's user assigned principal id:
 ```bash
-az vmss identity show -g <resource group>  -n <vmss scalset name> -o yaml
+az aks show -g <resource group> -n <aks cluster name> --query identityProfile.kubeletidentity.clientId -o tsv
```

Comment:
This command works for AKS clusters, but we also need to keep the `az vmss identity show` command for aks-engine created clusters. So can we keep the old command and also this command for AKS clusters?

Reply:
Applied these changes, let me know what you think!